### ISSUE TITLE
What is the input and target for the loss function?

### ISSUE THREAD

**Issue by sarmientoj24** [pangwong/pytorch-multi-label-classifier#10]
Say I have 5 classes, what would be the input and target?
